3_run_rsync/vars/defaults.yml

```diff
@@ -0,0 +1,14 @@
+# Stage 3 Input
+pvc_data_filepath: ../output/pvc-data.json
+namespace_data_filepath: ../output/namespace-data.json
+
+# Stage 3 Output
+failed_pvcs_dir: ../output/
+successful_pvcs_dir: ../output/
+
+# Recommended Transfer Pod Image
+transfer_pod_image: "quay.io/konveyor/ssh-transfer:latest"
```

Comment:
Let's change this to `quay.io/konveyor/stunnel-ssh-transfer:latest`. I set up a new repo and disabled the rebuild on the old ssh-transfer repo so we don't brake Experian when we merge this.